libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1234,6 +1388,69 @@ pub fn find_complete_timing_marks_from_partial_timing_marks(
         );
     }
 
+    const MIN_REQUIRED_CORNER_MARK_SCORE: f32 = 0.5;
```

Comment:
How did you pick this threshold? What's the general range of scores you've seen?

Reply:
I mostly just left it as it was, though I realize that the thing being scored is different. In practice, for clearly defined timing marks with a small amount of noise around them, the score is 60-90%:

<img width="490" alt="image" src="https://github.com/user-attachments/assets/a25cc897-2418-4eba-b650-7c2cdd7df6b6">

![image](https://github.com/user-attachments/assets/cdc39c99-bce2-4f0f-9acb-665021bd4a14)

![image](https://github.com/user-attachments/assets/2bb409b9-6091-43c6-8beb-d8d444da0045)

Reply:
Ok, great!